src/odinapi/templates/index.html

```diff
@@ -28,32 +28,12 @@ <h2 class="featurette-heading">Documentation updated<span class="text-muted"> 20
 
           <p>All documents can be found <a href="#documents"> here </a></p>
         </div>
-        <div class="col-md-5">
```

Comment:
Was it by intention that the image was removed too?

Reply:
Yes. It was ugly, so I decided to keep the one used for the caution.